class-wcal-update.php

```diff
@@ -289,6 +289,10 @@ public static function wcal_alter_tables( $db_prefix, $blog_id ) {
 			if ( ! $wpdb->get_var( 'SHOW COLUMNS FROM `' . $db_prefix . "ac_sent_history_lite` LIKE 'encrypt_key';" ) ) { //phpcs:ignore
 				$wpdb->query( 'ALTER TABLE ' . $db_prefix . 'ac_sent_history_lite ADD `encrypt_key` VARCHAR(500) NOT NULL AFTER `sent_email_id`;' ); //phpcs:ignore
 			}
+			// 5.20.1
+			if ( ! $wpdb->get_var( 'SHOW COLUMNS FROM ' . $db_prefix . "ac_email_templates_lite LIKE 'email_type'" ) ) { //phpcs:ignore
```

Comment:
@nidhitatosaniya Please use the collation for the system. Please update other SQL code where the system collation has not been used.

Reply:
it's already updated see here : https://github.com/TycheSoftwares/woocommerce-abandoned-cart/pull/988/files#diff-d96aaccab96ab9996d0f1e8d628fe02c4fa4f6af462189201d20b5527d83ba50R294